### PR TITLE
feat: NTRN-362. Implement the credits vault contract tests

### PR DESCRIPTION
**Pull request by albertandrejev**
We want to have another vault for cNTRNs locked in the credits contract. After the end of the TGE, while vesting and until claimed, tokens in the credits contract have voting power. We don’t need the vault to allow users to Add / Remove tokens; the only thing we want from the vault is to check the user balance(s) in the Credits contract and give cNTRNs the same voting power that regular neutrons have.

https://p2pvalidator.atlassian.net/browse/NTRN-362

Merge with:
- https://github.com/neutron-org/neutron-dao/pull/61

### CODE REVIEW

File: src/testcases/parallel/credits.test.ts [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+  let cm_dao: CosmosWrapper;
+  let cm_manager: CosmosWrapper;
+  let dao_wallet: Wallet;
+  let manager_wallet: Wallet;
+  let airdrop_wallet: Wallet;
+  let lockdrop_wallet: Wallet;
+  let dao_addr: AccAddress | ValAddress;
+  let airdrop_addr: AccAddress | ValAddress;
+  let lockdrop_addr: AccAddress | ValAddress;
+  let manager_addr: AccAddress | ValAddress;
```
Comment (ratik) [inline src/testcases/parallel/credits.test.ts:22]: do you mind renaming to camelCase here and the following?

**Comment by albertandrejev** [comment]
[Tests run](https://github.com/neutron-org/neutron-tests/actions/runs/4719734312)

**Comment by foxpy** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/4893042867

**Comment by foxpy** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/4894823361

**Comment by foxpy** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/4895468457

**Comment by foxpy** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/4896024451

**Pull request merged**